### ISSUE TITLE
UI is blocked for Pull Requests with many comments

### ISSUE THREAD

**Issue by jazzbox** [CodeHubApp/CodeHub#372]
When loading a pull request like https://github.com/apple/swift/pull/12442 the UI is blocked for many seconds, the App is completely unresponsive in this time.

I suppose that the network request is done on the main and not in a background thread.

**Comment by thedillonb** [comment]
Holy cow - that's a big thread. The network-request is actually done on a worker thread but the UI rendering and markdown parsing is not. I'll reproduce this on my setup and see what I can do about it. Thanks for spotting this.

**Comment by jazzbox** [comment]
Yeah, I checked it again and it seems that only PRs with comments from swift-ci are slow.

This PR https://github.com/apple/swift/pull/13058 is another strange case, it shows huge user icons on my iPhone!

**Comment by jazzbox** [comment]
Believe it or not, but the PR I mentioned first is still alive and growing. CodeHub is now either crashing or is killed by the system for being unresponsive so long (I guess the latter).

Unfortunately there is no other way to uncheck this PR in the notification view as to click on it. And even if you immediately leave the view, the app gets stuck in whatever other view.
